common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3228,6 +3229,14 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(3)
 
+  val CLIENT_FETCH_THROWS_FETCH_FAILURE: ConfigEntry[Boolean] =
+    buildConf("celeborn.client.spark.fetch.throwsFetchFailure")
+      .categories("client")
+      .version("0.4.0")
```

Comment:
```suggestion
      .version("0.3.2")
```

Reply:
Hi @SteNicholas , since this is a big feature, I suggest not merge to branch-0.3